packages/ui-admin/package.json

```diff
@@ -58,15 +60,18 @@
     "@types/react-select": "^2.0.19",
     "cross-env": "^5.2.0",
     "css-loader": "^5.2.1",
+    "style-loader": "1.3.0",
     "css-modules-typescript-loader": "^4.0.1",
+    "filemanager-webpack-plugin": "3.1.1",
     "expose-loader": "^1.0.3",
     "hard-source-webpack-plugin": "git+https://github.com/botpress/hard-source-webpack-plugin.git",
     "http-proxy-middleware": "^1.2.0",
     "node-sass": "^4.14.1",
     "node-sass-chokidar": "^1.3.4",
     "npm-run-all": "^4.1.5",
     "react-app-rewired": "^2.1.8",
-    "react-scripts": "^4.0.3"
+    "react-scripts": "^4.0.3",
+    "typescript": "^4.4.4"
```

Comment:
I'm working on transitionning the repo to yarn2, so while using the yarn doctor, it flags missing dependencies in packages. I think it could be useful to list dependencies used by the package, even if when using yarn workspaces, we have a single node_modules so we only have one copy of TS for all of them